### ISSUE TITLE
Accessibility - The purpose of each link should be clear from the link text (4)

### ISSUE THREAD

**Issue by samatar26** [FAC-11/week1-BAJA#56]
Providing a descriptive text label lets a user distinguish the link from links in the Web page that lead to other destinations and helps the user determine whether to follow the link.  In some assistive technologies the aria-label value will show in the list of links instead of the actual link text.
Resources: https://www.w3.org/WAI/GL/wiki/Using_aria-label_for_link_purpose
